db/schema.rb

```diff
@@ -11,11 +11,35 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160622075431) do
+ActiveRecord::Schema.define(version: 20160623172311) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "booking_requests", force: :cascade do |t|
+    t.string   "location_id",                null: false
+    t.string   "name",                       null: false
+    t.string   "email",                      null: false
+    t.string   "phone",                      null: false
+    t.string   "memorable_word",             null: false
+    t.string   "age_range",                  null: false
+    t.boolean  "accessibility_requirements", null: false
+    t.boolean  "marketing_opt_in",           null: false
+    t.boolean  "defined_contribution_pot",   null: false
+    t.datetime "created_at",                 null: false
+    t.datetime "updated_at",                 null: false
+  end
+
+  create_table "slots", force: :cascade do |t|
+    t.integer  "booking_request_id"
```

Comment:
Oh, that's weird. I was sure the `belongs_to` in a migration adds that already. I'll sort it.